timesketch/frontend-ng/src/views/Explore.vue

```diff
@@ -260,6 +260,64 @@ limitations under the License.
       </div>
     </div>
 
+    <!-- write a v-dialog here that shows the shortcut help -->
+    <v-dialog v-model="showShortcutHelp" width="700" style="overflow: visible">
+      <v-card width="700" style="overflow: visible">
+        <v-card-title> Keyboard Shortcut help </v-card-title>
+
+        <v-card-text
+          ><v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> Ctrl </v-btn>
+          +
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> Shift </v-btn>
+          +
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> n </v-btn>
+          = Next page</v-card-text
+        >
+
+        <v-card-text>
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> Ctrl </v-btn>
+          +
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> Shift </v-btn>
+          +
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> p </v-btn>
+          = Previous page
+        </v-card-text>
+
+        <v-card-text v-shortkey="['ctrl', 'shift', 't']" @shortkey="toggleSearchHistory()">
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> Ctrl </v-btn>
+          +
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> Shift </v-btn>
+          +
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> t </v-btn>
+          = Toggle Search History
+        </v-card-text>
+        <v-card-text>
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> Ctrl </v-btn>
+          +
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> Shift </v-btn>
+          +
+          <v-btn style="text-transform: unset; cursor: unset" :ripple="false" x-small density="compact"> h </v-btn>
+          = Toggle Shortcut help
+        </v-card-text>
+        <v-card-text v-shortkey="['ctrl', 'shift', 'm']" @shortkey="addEventBtn()">
```

Comment:
This function `addEventBtn()` is still missing. Either implement the feature or mark the shortcut as not active yet.

Reply:
just noticed it as well, changed, now works